huaweicloud/services/iec/data_source_huaweicloud_iec_flavors.go

```diff
@@ -73,12 +76,12 @@ func dataSourceIecFlavors() *schema.Resource {
 	}
 }
 
-func dataSourceIecFlavorsV1Read(d *schema.ResourceData, meta interface{}) error {
-	config := meta.(*config.Config)
+func dataSourceIecFlavorsRead(d *schema.ResourceData, meta interface{}) error {
+	cfg := meta.(*config.Config)
 
-	iecClient, err := config.IECV1Client(GetRegion(d, config))
+	iecClient, err := cfg.IECV1Client(common.GetRegion(d, cfg))
 	if err != nil {
-		return fmtp.Errorf("Error creating HuaweiCloud IEC client: %s", err)
+		return fmt.Errorf("error creating iec client: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("error creating IEC client: %s", err)
```
`IEC` is an abbreviation and needs to be capitalized, please check the other parts